### DISCUSSION TITLE
"Dependant" Tab or Search

### DISCUSSION THREAD

**Discussion by IdrisQe** [modrinth/.github#134] [Feature requests]
Having dependencies show on each file is useful, but it's a bit awkward to find mods that use a library the other way around (like how you can do in Curseforge's Dependants Tab) - it would be nice to have a way to check this. Either through a tab like Curseforge has, or by adding a "dependency" box in the search where you can put a mod's URL ID, showing only mods that have files for the selected search version/loader that also have the input mod as a dependency.

For example if I put "fabric-api" in the dependency box, then selected 1.19 for version and Quilt for loader, it would show only mods that have a file that: needs Fabric API, uses Quilt Loader, and is for Minecraft 1.19

I might be weird for using this a lot, I just find when I need to add a Library mod for one mod, I like to see what other use I can get out of it.

**Comment by triphora** [comment]
Duplicate of #90

**Comment by IdrisQe** [comment]
Whoops, this didn't appear when I was trying to find similar topics, sorry
